### PR TITLE
Bump bitcoin dependency to 0.25

### PR DESCRIPTION
**Pull request by romanz**
Not sure about this crate versioning - should it be also bumped (to 0.12)?

### CODE REVIEW

File: integration_test/src/main.rs [hunk 1]
```diff
@@ -258,8 +257,8 @@ fn test_get_block_header_get_block_header_info(cl: &Client) {
     let tip = cl.get_best_block_hash().unwrap();
     let header = cl.get_block_header(&tip).unwrap();
     let info = cl.get_block_header_info(&tip).unwrap();
-    assert_eq!(header.bitcoin_hash(), info.hash);
-    assert_eq!(header.version, info.version);
+    assert_eq!(header.block_hash(), info.hash);
+    assert_eq!(header.version as u32, info.version);
```
Comment (RCasatta) [inline integration_test/src/main.rs:261]: maybe better to change info.version to i32?

Comment (stevenroose) [reply integration_test/src/main.rs:261]: Yeah I agree with this. Bumping bitcoin is a major semver bump here anyway too, so it doesn't matter if we have to change fields.

**Review by stevenroose — CHANGES_REQUESTED** [review]
Can you also make sure this is rebased on master?

File: integration_test/src/main.rs (hunk 1 above)
Comment (romanz) [reply integration_test/src/main.rs]: Sounds good - changed at https://github.com/rust-bitcoin/rust-bitcoincore-rpc/pull/141/commits/a24d6089b17a36536af2048da6801f9d9a006d1b.

**Comment by romanz** [comment]
> Can you also make sure this is rebased on master?

Rebased and force-pushed 637e895.

**Review by stevenroose — APPROVED** [review]

**Comment by romanz** [comment]
Thanks!

**Pull request merged**